### PR TITLE
Update email-activity.md

### PR DESCRIPTION
**Pull request by Divya-RP**
Add the licensing requirement information to the report

### CODE REVIEW

**Comment by mypil** [comment]
@kwekuako - PR is ready for final review. Thank you.

**Comment by cmcatee-MSFT** [comment]
@Divya-RP I don't understand the note you added. You said it's licensing requirement information, but the note says the email activity report is available for licensed user mailbox. How is that a requirement? Can you explain why this note is needed?

**Comment by Divya-RP** [comment]
Hell0 Cern,

The article does not talk about the license requirement. The email activity report is available only for mailboxes which have licenses.

Example: Shared mailbox does not have license, the email activity report is not available for the shared mailbox

Regards,
Divya

File: Office365-Admin/activity-reports/email-activity.md [hunk 1]
```diff
@@ -61,4 +61,4 @@ You can get a view into your user's email activity by looking at the **Activity*
 |8.  <br/> |You can also export the report data into an Excel .csv file, by clicking or tapping the **Export** ![Office 365 reports](../media/816a224b-6ca7-4967-a135-4f6427f64dc8.JPG) link. This exports data of all users and enables you to do simple sorting and filtering for further analysis. If you have less than 2000 users, you can sort and filter within the table in the report itself. If you have more than 2000 users, in order to filter and sort, you will need to export the data.  <br/> |
 |||
    
-
+Note: Email activity report is available for licensed user mailbox.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/activity-reports/email-activity.md]: > [!NOTE]
> The Email activity report is only available for mailboxes that are associated with users who have licenses.

Comment (mypil) [inline Office365-Admin/activity-reports/email-activity.md]:
```suggestion
Note: The Email activity report is only available for mailboxes that are associated with users who have licenses.
```

Comment (mypil) [reply Office365-Admin/activity-reports/email-activity.md]: @Divya-RP - please merge the commit on this PR following the suggestions made by @cmcatee-MSFT . Thank you.

**Comment by mypil** [comment]
@Divya-RP - We would like to follow up. Kindly merge the commit on this PR as suggested. Thank you.

**Comment by Divya-RP** [comment]
Hello Joyce,

I have committed the changes. Please let me know if anything else to be done.

Regards,
Divya

**Comment by mypil** [comment]
@cmcatee-MSFT - Your recommended change has been applied and PR is now ready for your final review. Please assist to merge the PR content. Thank you.

**Pull request merged**